mediatranslation/README.md

```diff
@@ -0,0 +1,50 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
```

Comment:
We need to decide if we want to use this readme and if we want to create sub-directories within the product directory to prepare for future (separate) samples

Reply:
Ack.